packages/components/src/color-picker/styles.ts

```diff
@@ -42,14 +41,6 @@ export const RangeControl = styled( InnerRangeControl )`
 	}
 `;
 
-// All inputs should be the same height so this should be changed at the component level.
-// That involves changing heights of multiple input types probably buttons too etc.
-// So until that is done we are already using the new height on the color picker so it matches the mockups.
-const inputHeightStyle = `
-&&& ${ Input } {
-	height: 40px;
-}`;
-
```

Comment:
I'll go ahead and remove these 40px styles without moving them to a `large` size variant, since it's just a hack anyway. We can first add a `large` size variant to `InputControl` once we're ready to introduce large variants.